dev-haskell/hnix-store-remote/hnix-store-remote-0.7.0.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+#hackport: flags: -io-testsuite
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Remote hnix store"
+HOMEPAGE="https://github.com/haskell-nix/hnix-store"
+
+LICENSE="Apache-2.0"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="build-derivation build-readme"
+
+RDEPEND="dev-haskell/aeson:=[profile?]
+	dev-haskell/attoparsec:=[profile?]
+	dev-haskell/cereal:=[profile?]
+	dev-haskell/concurrency:=[profile?]
+	dev-haskell/crypton:=[profile?]
+	dev-haskell/data-default-class:=[profile?]
+	>dev-haskell/dependent-sum-0.7:=[profile?]
+	>=dev-haskell/dependent-sum-template-0.2.0.1:=[profile?] <dev-haskell/dependent-sum-template-0.3:=[profile?]
+	>=dev-haskell/dlist-1.0:=[profile?]
+	<dev-haskell/generic-arbitrary-1.1:=[profile?]
+	dev-haskell/hashable:=[profile?]
+	>=dev-haskell/hnix-store-core-0.8:=[profile?] <dev-haskell/hnix-store-core-0.9:=[profile?]
+	>=dev-haskell/hnix-store-json-0.1:=[profile?]
+	>=dev-haskell/hnix-store-nar-0.1:=[profile?]
+	>=dev-haskell/hnix-store-tests-0.1:=[profile?]
+	dev-haskell/network:=[profile?]
+	dev-haskell/quickcheck:=[profile?]
+	dev-haskell/text:=[profile?]
+	dev-haskell/unordered-containers:=[profile?]
+	dev-haskell/vector:=[profile?]
+	>=dev-lang/ghc-9.0.2:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-3.4.1.0
+	test? ( dev-haskell/hspec
+		>dev-haskell/some-1.0.5 <dev-haskell/some-2 )
+	build-readme? ( dev-haskell/markdown-unlit )
+"
+
+src_configure() {
+	haskell-cabal_src_configure \
+		$(cabal_flag build-derivation build-derivation) \
+		$(cabal_flag build-readme build-readme) \
```

Comment:
```suggestion
		$(cabal_flag doc build-readme) \
```

Reply:
This isn't right. The `build-readme` flag builds a `remote-readme` example executable.

- [`hnix-store-remote.cabal`](https://hackage.haskell.org/package/hnix-store-remote-0.7.0.0/hnix-store-remote.cabal)
    ```cabal
    executable remote-readme
      if !flag(build-readme)
        buildable: False
    ```

Reply:
```diff
 src_configure() {
 	haskell-cabal_src_configure \
-		$(cabal_flag build-derivation build-derivation) \
-		$(cabal_flag build-readme build-readme) \
+		$(cabal_flag examples build-derivation) \
+		$(cabal_flag examples build-readme) \
```

This would make more sense to me. Also, you probably want to rename those executables using `CABAL_CHBINS` in the ebuild.

```ebuild
CABAL_CHBINS=(
	'build-derivation' "${PN}-build-derivation"
	'remote-readme' "${PN}-remote-readme"
)
```

Or, alternatively, just disable those flags altogether if the examples don't seem like they're worth much.